certs-gen/certs-gen.sh

```diff
@@ -129,10 +129,7 @@ extendedKeyUsage = serverAuth, clientAuth
 subjectAltName   = @alt_names
 # you will have to use the following DNS name and URI because it will be checked by the TSB
 [ alt_names ]
-DNS.1 = xcp.tetrate.io
-URI.1 = spiffe://xcp.tetrate.io/central
 DNS.2 = ${TSB_FQDN}
-DNS.3 = ${TSB_FQDN}:9443
 EOF
 
```

Comment:
Technically we can call certs-gen.sh - generate rootCA or can we actually get rid of it completely as in theory tsb provided certs are self-signed anyway? so technically we don't need the script anymore?

Reply:
I left the generate to allow us to use FQDN in certificate, as it looks cert-management currently will create`certificate` based on status of service and doesn't have a way for us to provide SAN property.

The other piece is we need to get CA certs to controlplane as these wont get passed, while we can use what TSE does:
`tctl apply -f "$FOLDER/cluster-$CLUSTER_NAME.yaml" -o yaml | yq '.spec.installTemplate.helm' > "$FOLDER/$CLUSTER_NAME-controlplane_values.yaml"`
But the problem with this is we impose the `yq` tool where for this repo we were aiming to use built in tooling.

WDYT @smarunich